frontend/src/components/USWDSIcon.tsx

```diff
@@ -11,6 +11,8 @@ interface IconProps {
 // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
 const sprite_uri = SpriteSVG.src as string;
 
+// height prop doesn't seem to work
```

Comment:
Seems like this could be discussion rather than a code comment. The USWDS component doesn't work with SVGs as far as I recall. We could revisit that, was a while ago this was adopted.

Reply:
true, I'll remove the comment. Ticket for follow up here: https://github.com/HHS/simpler-grants-gov/issues/3748